graphql-relay/interface/presenter/card.go

```diff
@@ -54,14 +55,14 @@ func NewCardStatus(in entity.CardStatus) model.CardStatus {
 	}
 }
 
-func NewCard(c *entity.Card) *Card {
+func ToCard(c *entity.Card) *Card {
 	return &Card{c}
 }
 
-func NewCardNodes(cs entity.Cards) []node {
-	res := make([]node, 0, len(cs))
+func ToCards(cs []*entity.Card) []*Card {
```

Comment:
`entity.Cards` だと問題あるのでしょうか？

Reply:
Sorry, I overlooked this. if there is a type alias, it would be better to use that. I have fixed it.
https://github.com/Reimei1213/lab/pull/1/commits/f9c8260faa43581628f91dbf9d23ddcd54d3bc96